session.go

```diff
@@ -252,11 +257,20 @@ func (s *Session) RunAnalyzeQuery(stmt spanner.Statement) (*pb.QueryPlan, error)
 
 func (s *Session) runQueryWithOptions(stmt spanner.Statement, opts spanner.QueryOptions) (*spanner.RowIterator, *spanner.ReadOnlyTransaction) {
 	if s.InReadWriteTransaction() {
+		if s.tc.transactionTag != "" {
+			// In a read-write transaction, add transaction tags as request tags.
+			opts.RequestTag = s.tc.transactionTag
+		}
```

Comment:
Go string's zero value is empty string `""`, so we can just put `s.tc.transactionTag` into `ops.RequestTag` as follows:

```suggestion	
		opts.RequestTag = s.tc.transactionTag
```

Reply:
Thanks for pointing that out. I have fixed it.